Gems/Vegetation/Code/Source/Components/MeshBlockerComponent.cpp

```diff
@@ -21,6 +21,10 @@
 #include <Vegetation/InstanceData.h>
 #include <Atom/RPI.Reflect/Model/ModelAsset.h>
 
+#include "../VegetationProfiler.h"
```

Comment:
Nit: Can this just be <VegetationProfiler.h>?

Reply:
@amzn-tommy made this suggestion, I already, i must have missed this one